go-controller/.golangci.yml

```diff
@@ -28,6 +29,23 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
+  
+  importas:
+    no-unaliased: true
+    alias:
+      # Kubernetes
+      - pkg: k8s.io/api/core/v1
+        alias: corev1
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
+        alias: metav1
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: apierrors
```

Comment:
nit: kapierrors

Reply:
I am aligning all these imports as they are in other kubernetes projects (example: https://github.com/kubernetes-sigs/controller-runtime/blob/d6b34400308c42fee4aa23a751eaa1f1f157eec4/.golangci.yml#L52) . However, the actual nomenclature itself is not the most important thing, rather that it's homogeneous throughout the project.

Reply:
agree that I care more about homogeneousness than what's its actually called. was just a nit